spark/src/main/java/org/opensearch/sql/spark/transport/TransportGetQueryResultRequestAction.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.sql.spark.transport;
+
+import org.opensearch.action.ActionType;
+import org.opensearch.action.support.ActionFilters;
+import org.opensearch.action.support.HandledTransportAction;
+import org.opensearch.common.inject.Inject;
+import org.opensearch.core.action.ActionListener;
+import org.opensearch.sql.spark.transport.model.GetJobQueryResultActionRequest;
+import org.opensearch.sql.spark.transport.model.GetJobQueryResultActionResponse;
+import org.opensearch.tasks.Task;
+import org.opensearch.transport.TransportService;
+
+public class TransportGetQueryResultRequestAction
+    extends HandledTransportAction<
+        GetJobQueryResultActionRequest, GetJobQueryResultActionResponse> {
+
+  public static final String NAME = "cluster:admin/opensearch/ql/jobs/result";
```

Comment:
what is the different of result and status? result API include status, right?

Reply:
yes. Something like below.
```
{
      "status" : "IN_PROGRESS"
}
```
```
{
      "status" : "COMPLETED",
      "schema" : {},
      "result" : {}
}
```